backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -567,15 +567,16 @@ private OnServerResponseEventListener getDatabaseCallback(final BacktraceDatabas
         return new OnServerResponseEventListener() {
             @Override
             public void onEvent(BacktraceResult backtraceResult) {
-                if (customCallback != null) {
-                    customCallback.onEvent(backtraceResult);
-                }
                 if (record != null) {
                     record.close();
                 }
                 if (backtraceResult != null && backtraceResult.status == BacktraceResultStatus.Ok) {
                     database.delete(record);
                 }
+
```

Comment:
why is this part of this PR?

Reply:
This was the initial ask of this pull request, but later I recognized it might be related to the invalid token. it would be great if we can leave it here, but if you prefer to have a separated diff for it - I'm ok with it and I will create a new one.

Reply:
👍